src/jdk.jfr/share/classes/jdk/jfr/consumer/RecordedThread.java

```diff
@@ -97,7 +97,7 @@ public long getJavaThreadId() {
      * reused within the lifespan of the JVM.
      * <p>
      * See {@link #getJavaThreadId()} for the ID that is returned by
-     * {@code java.lang.Thread.getId()}
+     * {@code java.lang.Thread.threadId()}
```

Comment:
Looks okay but maybe it should be a link?

Reply:
I considered it, but the purpose of the comment is to help users not mix up getId() with getJavaThreadId() in the RecordedThread class. If users follows a link to java.lang.Thread and read about the two IDs there, it might confuse them. The difference between getId() and threadId() in j.l.Thread is not the same as in RecordedThread.

That said, if you think a link would be helpful I can add it.

Reply:
Thanks for the feedback. I added a link Thread::threadId() to RecordedThread::getJavaThreadId(), but didn't link RecordedThread::getId(). I also added a link to getOSThreadId() from getId().